src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -169,11 +169,16 @@ export class TxSender {
       return
     }
 
-    const contractErrorMessage = await getContractErrorMessage({
-      safeInstance,
-      from,
-      data: executeData,
-    })
+    let contractErrorMessage
+    try {
+      contractErrorMessage = await getContractErrorMessage({
+        safeInstance,
+        from,
+        data: executeData,
+      })
+    } catch (err) {
+      notifications.showOnError(err, err.message)
```

Comment:
You could log the error here instead of in the decoding function.

Reply:
Yes, that makes more sense. Will move the log here